### DISCUSSION TITLE
Office_S_Tips/pdf_docx/2019/09/23/z_PDF_do_DOCX

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#24] [Komentarze]
# Office_S_Tips/pdf_docx/2019/09/23/z_PDF_do_DOCX

Rysunki wektorowe i inne fragmenty PDF można skopiować jako obrazy do Worda w niezłej jakości.

https://andrzejq.github.io/Office_S_Tips/pdf_docx/2019/09/23/z_PDF_do_DOCX.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
